src/test/java/com/auth0/exception/APIExceptionTest.java

```diff
@@ -13,16 +14,85 @@
 
     private Map<String, Object> values;
 
+    private final static int ERROR_CODE = 42;
+    private final static String EXPECTED_ERROR_MESSAGE_PREFIX = "Request failed with status code " + ERROR_CODE + ": ";
+
     @Before
     public void setUp() {
         values = new HashMap<>();
     }
 
+    @Test
+    public void shouldGetMessageFromErrorDescription() {
+        values.put("error_description", "some error description");
+        values.put("description", "some description");
+        values.put("message", "some message");
+        values.put("error", "some error");
+        APIException apiException = new APIException(values, ERROR_CODE);
```

Comment:
the ERROR_CODE value is never checked later. Maybe add one test for asserting it gets set somewhere?

Reply:
it is checked, through the `EXPECTED_ERROR_MESSAGE_PREFIX` constant above.

Reply:
Correct. I thought there was a getter for the error code as well.